packages/mobile/src/identity/securityCode.ts

```diff
@@ -78,15 +78,16 @@ async function requestValidator(
       securityCode,
     }
 
-    return attestationsWrapper.getAttestationForSecurityCode(
+    const attestationCode = await attestationsWrapper.getAttestationForSecurityCode(
       attestation.attestationServiceURL,
       requestBody,
       signer
     )
+    return attestationCode
   } catch (error) {
     Logger.error(
       TAG + '@getAttestationCodeFromSecurityCode',
-      `get for issuer ${issuer} failed`,
+      `get for issuer ${issuer} failed - ${JSON.stringify(error)}`,
```

Comment:
Don't think this change is necessary anymore. `Logger.error` should be handing `error` correctly now